jdaviz/configs/default/plugins/export_plot/export_plot.py

```diff
@@ -1,23 +1,60 @@
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import PluginTemplateMixin, ViewerSelectMixin
+from jdaviz.core.user_api import PluginUserApi
 
 __all__ = ['ExportViewer']
 
 
 @tray_registry('g-export-plot', label="Export Plot")
 class ExportViewer(PluginTemplateMixin, ViewerSelectMixin):
+    """
+    See the :ref:`Export Plot Plugin Documentation <imviz-export-plot>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * :meth:`save_figure`
+    """
     template_file = __file__, "export_plot.vue"
 
+    @property
+    def user_api(self):
+        return PluginUserApi(self, expose=('save_figure',))
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def vue_save_figure(self, filetype):
+    def save_figure(self, filename=None, filetype=None):
         """
-        Callback for save figure events in the front end viewer toolbars. Uses
-        the bqplot.Figure save methods.
+        Save the figure to an image with a provided filename or through an interactive save dialog.
+
+        Parameters
+        ----------
+        filename : str or None
+            Filename to autopopulate the save dialog.
+        filetype : str or None

```

Comment:
When there are specific valid options, better to list them out explicitly.

```suggestion
        filetype : {'png', 'svg', `None`}
```